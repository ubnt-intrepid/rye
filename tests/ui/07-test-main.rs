fn main() {}

mod unexpected_path_patterns {
    rye::test_main! {
        a,
        b::c,
        {d, e, f},
        *,
        foo as bar,
    }
}

mod unresolved_test_path {
    rye::test_main! {
        a,
        b::c,
        {d, e},
        f::g::{h, i},
    }
}
