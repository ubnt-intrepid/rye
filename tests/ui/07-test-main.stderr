error: glob pattern is forbidden
 --> $DIR/07-test-main.rs:8:9
  |
8 |         *,
  |         ^

error: rename pattern is forbidden
 --> $DIR/07-test-main.rs:9:9
  |
9 |         foo as bar,
  |         ^^^^^^^^^^

error[E0433]: failed to resolve: use of undeclared type or module `b`
  --> $DIR/07-test-main.rs:16:9
   |
16 |         b::c,
   |         ^ use of undeclared type or module `b`

error[E0433]: failed to resolve: use of undeclared type or module `f`
  --> $DIR/07-test-main.rs:18:9
   |
18 |         f::g::{h, i},
   |         ^ use of undeclared type or module `f`

error[E0425]: cannot find value `a` in this scope
  --> $DIR/07-test-main.rs:15:9
   |
15 |         a,
   |         ^ not found in this scope

error[E0425]: cannot find value `d` in this scope
  --> $DIR/07-test-main.rs:17:10
   |
17 |         {d, e},
   |          ^ not found in this scope

error[E0425]: cannot find value `e` in this scope
  --> $DIR/07-test-main.rs:17:13
   |
17 |         {d, e},
   |             ^ not found in this scope
